src/web/components/lib/api.tsx

```diff
@@ -1,30 +1,40 @@
 import { useState, useEffect } from 'react';
 
+interface FetchOptions {
```

Comment:
🎖 This is strict but better that and force people to update this type to do anything different than letting them pass anything